core/storage/audit/gae_models_test.py

```diff
@@ -57,6 +57,24 @@ def test_get_deletion_policy(self) -> None:
             audit_models.RoleQueryAuditModel.get_deletion_policy(),
             base_models.DELETION_POLICY.KEEP)
 
+    def test_get_export_policy(self) -> None:
+        sample_dict = base_models.BaseModel.get_export_policy()
+        sample_dict.update(
+            {'user_id': base_models.EXPORT_POLICY.NOT_APPLICABLE,
```

Comment:
Put the { at the end of the previous line. Ditto elsewhere.

Reply:
Changed the test, hence ommitted.